sig-creation-procedure.md

```diff
@@ -0,0 +1,85 @@
+### SIG creation procedure
+
+#### 
+**Google Groups creation**
+
+Create Google Groups at [https://groups.google.com/forum/#!creategroup](https://groups.google.com/forum/#!creategroup), following the procedure: 
+
+* Create a group using the name convention below;
+* Group has to be created as an e-mail list with at least three owners (including sarahnovotny at google.com and ihor.dvoretskyi at gmail.com);
+* To add the owners, visit the Group Settings (drop-down menu on the right side), select Direct Add Members on the left side and add Sarah and Ihor via email address (with a suitable welcome message); in Members/All Members select Ihor and Sarah and assign to an "owner role";
+* Set "View topics", "Post", "Join the Group" permissions to be "Public"
+
+
+Name convention:
+
+* kubernetes-sig-foo-misc
```

Comment:
Having discussed this question with @bgrant0607 we've decided to avoid having non-suffixed groups.

If you have any suffix changing proposal (instead of *-misc), let's discuss it here.